kibana/README.md

```diff
@@ -64,12 +65,17 @@ helm install --name kibana elastic/kibana --set imageTag=7.3.0
 | `tolerations`             | Configurable [tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)                                                                                                                                                                                                                                                            | `[]`                                                                                                                      |
 | `ingress`                 | Configurable [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) to expose the Kibana service. See [`values.yaml`](./values.yaml) for an example                                                                                                                                                                                       | `enabled: false`                                                                                                          |
 | `service`                 | Configurable [service](https://kubernetes.io/docs/concepts/services-networking/service/) to expose the Kibana service. See [`values.yaml`](./values.yaml) for an example                                                                                                                                                                                       | `type: ClusterIP`<br>`port: 5601`<br>`nodePort:`<br>`annotations: {}`                                                     |
 | `labels`                  | Configurable [label](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) applied to all Kibana pods                                                                                                                                                                                                                                     | `{}`                                                                                                                      |
+| `dashboardImport.enabled`                  | Enable dashboard import                                                | `false`                               |
+| `dashboardImport.timeout`                  | Time in seconds waiting for Kibana to be in green overall state        | `60`                                  |
+| `dashboardImport.xpackauth.enabled`        | Enable Xpack auth                                                      | `false`                               |
+| `dashboardImport.xpackauth.username`       | Optional Xpack username                                                | `myuser`                              |
+| `dashboardImport.xpackauth.password`       | Optional Xpack password                                                | `mypass`                              |
+| `dashboardImport.dashboards`               | Dashboards                                                             | `{}`                                  |
```

Comment:
Could you explain what formats are supported? From the import script I can see that you can specify http URLs or files that are mounted within the initContainer.